apis/guides/Current Member Data.md

```diff
@@ -29,10 +29,10 @@ GET  https://{site_domain}/_api/apps/current-member/{app_id}
 
 |Name|Description|
 |---|---|
-|**member**|Member data|
-| member.**id** | GUID of the user that can be used to accessed more information|
-| member.**role**| Site visitor's role: “MEMBER”|
-| member.**email**| Email address|
+|**member**|Member data object|
+| member.**id** | GUID of the user that can be used to access more information|
+| member.**role**| Site visitor's role. Can be one of: “MEMBER” if he is a logged in site memeber, "OWNER" if the user is the site owner or "CONTRIBUTOR" if the user is one the site team members|
```

Comment:
Site visitor's role. One of: MEMBER/OWNER/CONTRIBUTOR
(no need for the detailed descriptions)